src/Perl6/Actions.nqp

```diff
@@ -1886,7 +1886,7 @@ class Perl6::Actions is HLL::Actions does STDActions {
         # isn't already declared.
         my $BLOCK := $*W.cur_lexpad();
         my $type := $BLOCK.ann('IN_DECL');
-        if $type eq 'mainline' && %*COMPILING<%?OPTIONS><setting> eq 'NULL' {
+        if $type eq 'mainline' && %*COMPILING<%?OPTIONS><setting> eq 'NULL.c' {
```

Comment:
I don't see why `NULL.c`. `NULL` is simply the absence of a setting; I don't see how nothing can be versioned.

Reply:
Currently `NULL.d` and `NULL.e` are used for building corresponding cores. `ModuleLoader` uses this revisions to find and pre-load previous `CORE`s. `NULL.c` is to unify building of all three cores in the build subsystem. Otherwise it requires a separate rule in `Makefile.in` template.

Reply:
> Currently NULL.d and NULL.e are used for building corresponding cores.

Hmm, that's odd, in that I thought CORE.d would just use CORE.c as its setting, and CORE.e would use CORE.d as its setting.